package.json

```diff
@@ -13,6 +13,7 @@
   },
   "dependencies": {
     "@nextui-org/react": "^2.1.13",
+    "@phosphor-icons/react": "^2.0.15",
```

Comment:
We need to remove the `react-icons` (line 25) package if we want to use this one.
I'm definitely happy to use this but let's try to not have multiple packages in the project otherwise it might become a mess

Reply:
Yep - didn't notice.